rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -1259,6 +1263,24 @@ def wait(self,timeout = None):
 		retcode = connector_binding.wait(self.native,timeout)
 		_check_retcode(retcode)
 
+	@staticmethod
+	def set_max_objects_per_thread(value):
+		"""Allows increasing the number of Connector instances that can be created
+
+		The default value is 1024. If your application creates more than eight
+		``Connector`` instances approximately, you may have to increase this
+		value.
+
+		This operation can only be called before creating any ``Connector``
```

Comment:
Does sphinx understand @staticmethod? If not we should clarify that here (or move it out of the Connector class)